lib/screens/profile.dart

```diff
@@ -0,0 +1,272 @@
+import 'dart:convert';
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:hive/hive.dart';
+import 'package:mvp/classes/prefrenses.dart';
+import 'package:mvp/constants/apiCalls.dart';
+import 'package:mvp/constants/themeColours.dart';
+import 'package:mvp/screens/auth/login.dart';
+import 'package:mvp/screens/common/cartIcon.dart';
+import 'package:mvp/screens/location.dart';
+import 'package:http/http.dart' as http;
+import 'package:mvp/screens/orders/ordersScreen.dart';
+import 'package:mvp/sizeconfig/sizeconfig.dart';
+
+class Profile extends StatefulWidget {
+  final String username;
+  final String mobile;
+  final String email;
+  Profile({this.username, this.mobile, this.email});
```

Comment:
we are using all of these constructor parameters, `@required` ?

Reply:
Will add required